models/public/efficientdet-d0-tf/model.yml

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The "efficientdet-d0-tf" model is one of the EfficientDet <https://arxiv.org/abs/1911.09070>
+  models  designed to perform object detection. This model was pretrained in TensorFlow*.
+  All the EfficientDet models have been pretrained on the MSCOCO* image database.
+  For details about this family of models, check out the Google AutoML repository
+  <https://github.com/google/automl/tree/master/efficientdet>.
+task_type: detection
+files:
+  - name: efficientdet-d0.tar.gz
+    size: 28828936
+    sha256: 74794c937aa1fa2f559c2393d22b251b1f7135a49b108bd0414bc4f4800ca15d
+    source: https://storage.googleapis.com/cloud-tpu-checkpoints/efficientdet/coco2/efficientdet-d0.tar.gz
+  - name: model_inspect.py
+    size: 20388
+    sha256: 6c68fe02f10d62dd87c2fc550b41c3df5cce52f0449be22e0699fbf209e3cbc1
+    source: https://raw.githubusercontent.com/google/automl/341af7d4da7805c3a874877484e133f33c420ec5/efficientdet/model_inspect.py
```

Comment:
Please place all source files in a subdirectory to avoid cluttering the download directory.

Reply:
Done